recipes/tqdm-multiprocess/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "tqdm-multiprocess" %}
+{% set version = "0.0.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # wheel...the setup.py file assumes a local requirements-dev.text
+  url: https://files.pythonhosted.org/packages/25/7e/0d889fc6c84e3df6b69aaafe893fc77f69b3d968ac9ce574d1c62c688050/tqdm_multiprocess-0.0.11-py3-none-any.whl
```

Comment:
Please update this to a pattern that looks similar to the one for sdists.

Reply:
Thanks @xhochy

This should have been marked as `draft` before, but is ready now for review.